index.html

```diff
@@ -35,11 +35,58 @@ <h1 class="header__title">TODO</h1>
         </span>
       </header>
       <main>
-        <!-- <input
-        type="text"
-        class="header__new-todo"
-        placeholder="Create a new todo..."
-      /> -->
+        <div class="new-todo">
+          <span class="new-todo__circle">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="25"
+              height="24"
+              viewBox="0 0 25 24"
+              fill="none"
+            >
+              <circle cx="12" cy="12" r="12" fill="white" stroke="#E3E4F1" />
+            </svg>
+          </span>
+          <input
+            type="text"
+            class="new-todo__input"
+            placeholder="Create a new todo..."
+          />
+        </div>
+        <div class="todo-list">
+          <div class="todo-list__item--checked">

```

Comment:
Como mencionado acima, aqui tu pode colocar o estilo base + o checked:

```suggestion
          <div class="todo-list__item todo-list__item--checked">
```